Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/Snapshots/FunctionsZipOutput_ToLower_Generated.g.cs

```diff
@@ -0,0 +1,71 @@
+// <auto-generated/>
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using Microsoft.Extensions.DependencyInjection;
+using Amazon.Lambda.Core;
+
+namespace TestServerlessApp.Sub1
+{
+    public class FunctionsZipOutput_ToLower_Generated
```

Comment:
Why are these files being added?
1. `FunctionsZipOutput_ToLower_Generated`
2. `ParameterlessMethodWithResponse_NoParameterWithResponse_Generated`
3. `ParameterlessMethods_NoParameter_Generated`


They seem unrelated to the new `OkResponseWithCustomSerializer`, so just trying to understand how/why they're here now.

Reply:
I had updated the snapshots programmatically. These may have been generated as a side effect. Deleted these files.